service/controller/aws/v17/version_bundle.go

```diff
@@ -17,6 +17,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Fix monitored file system mount points.",
 				Kind:        versionbundle.KindFixed,
 			},
+			{
+				Component:   "node-exporter",
+				Description: "Fix systemd collector D-Bus connection. https://github.com/giantswarm/kubernetes-node-exporter/pull/44",
```

Comment:
Does it make sense to track the PRs here? How does the official Changelog suggestion look like? Can we put links in braces when we need to track them? I would also like to have that specified in `fmt` then.

Reply:
I took formatting from here https://github.com/giantswarm/fmt/blob/master/releases/how_write_changelog_message.md#good-1

I think it's useful to know how this was fixed. But I won't insist on keeping this URL.

Reply:
I see it is described like that. All good.